diff --git a/node_modules/@adobe/gatsby-theme-aio/algolia/query-builder.js b/node_modules/@adobe/gatsby-theme-aio/algolia/query-builder.js
index faa9d6e..2fd432b 100644
--- a/node_modules/@adobe/gatsby-theme-aio/algolia/query-builder.js
+++ b/node_modules/@adobe/gatsby-theme-aio/algolia/query-builder.js
@@ -138,7 +138,7 @@ class QueryBuilder {
         minCharsLengthPerTag: 20,
         minWordsCount: 5
       };
-      records = await this.createRecordsForOpenApi.execute(node, options);
+      // records = await this.createRecordsForOpenApi.execute(node, options);
     } else {
       const options = {
         tagsToIndex: 'paragraph text, code, tableCell text',
